src/ui/FieldValue/IFieldValueCompatibleFacet.ts

```diff
@@ -6,6 +6,7 @@ export function isFacetFieldValueCompatible(facet: Component): facet is IFieldVa
 
 export interface IFieldValueCompatibleFacet extends Component {
   isFieldValueCompatible: boolean;
+  isFieldValueHierarchical: boolean;
```

Comment:
What would it take to be able to not have this value ?

Would we be able to derive it in another manner ?

Do you think that relying instead on `facet.type === 'DynamicHierarchicalFacet'` in `FieldValue.ts` could do the trick ?